hopsworks-api/src/main/java/io/hops/hopsworks/api/util/UploadService.java

```diff
@@ -287,7 +288,7 @@ private void confUploadTemplate() throws DatasetException {
     try {
       dfso = dfs.getDfsOps();
       if (!dfso.isDir(Settings.DIR_META_TEMPLATES)) {
-        dfso.mkdir(Settings.DIR_META_TEMPLATES);
+        dfso.mkdirs(new Path(Settings.DIR_META_TEMPLATES), FsPermission.getDefault());
```

Comment:
@o-alex what's the default?

Reply:
mkdir has this syntactic sugar method where you don't need to give permissions, while mkdirs doesn't.
so i checked what permission mkdir used and i used the same for the mkdirs